src/js/tracing/reactnativeprofiler.tsx

```diff
@@ -14,17 +14,23 @@ export class ReactNativeProfiler extends Profiler {
    */
   public componentDidMount(): void {
     super.componentDidMount();
-    getCurrentHub().getClient()?.addIntegration?.(createIntegration(this.name));
+    const client = getCurrentHub().getClient();
+    client && client.addIntegration && client.addIntegration(createIntegration(this.name));
```

Comment:
code looks good!

What about altering slightly the reading of the code, removing the else condition and storing the hub so we dont need to get the current hub twice:

```typescript
    const hub = getCurrentHub();
    const client = hub.getClient();

    if (!client) {
      // We can't use logger here because this will be logged before the `Sentry.init`.
      // eslint-disable-next-line no-console
      __DEV__ && console.warn('App Start Span could not be finished. `Sentry.wrap` was called before `Sentry.init`.');
      return;
    }

    client.addIntegration && client.addIntegration(createIntegration(this.name));
    const tracingIntegration = hub.getIntegration(
      ReactNativeTracing
    );

    tracingIntegration
      && this._mountSpan
      && typeof this._mountSpan.endTimestamp !== 'undefined'
      // The first root component mount is the app start finish.
      && tracingIntegration.onAppStartFinish(this._mountSpan.endTimestamp);
```

Reply:
Yes, looks good.